packages/api/src/repository/ResultRequest.ts

```diff
@@ -83,7 +83,8 @@ export class ResultRequestRepository {
         }
       }
     )
-    return this.normalizeId(lastResultRequest)
+
+    return lastResultRequest ? this.normalizeId(lastResultRequest) : null
```

Comment:
Why is this necessary?

Reply:
Because if `lastRequestResult` isn't found, it can't be normalized